packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -2608,6 +2612,13 @@ export class WalletService {
 
             async.series(
               [
+                (next) => {
+                  this.getMainAddresses({reverse: true, limit: 1}, (err, mainAddr) => {
```

Comment:
lets do this only on the chain that need it. Many add `needsFromAddress` flag in chain Service or something?

Reply:
added `isUTXOCoin` check